### PR TITLE
Configlet lint passing.

### PR DESCRIPTION
**Pull request by KalebHawkins**
This is a large pull request I apologize for that. To list what I've done in summary.

1. Updated the fetch-configlet script to match the latest (https://github.com/exercism/configlet/blob/main/scripts/fetch-configlet)
2. Corrected the exercise path in the run-tests.ps1 file.
3. Included exercises/shared/.docs directory with corresponding files to pass the configlet linting process.

### CODE REVIEW

**Comment by kotp** [comment]
Can you fix up the history of this so that the adding and removal of the binary files  are not in the history?

File: exercises/shared/.docs/debug.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Debug
```
Comment (ErikSchierboom) [inline exercises/shared/.docs/debug.md]: There is no [test runner](https://exercism.org/docs/building/tooling/test-runners) for powershell (see https://github.com/exercism/powershell-test-runner), which means that this document is not used. Could you remove it?

File: exercises/shared/.docs/cli.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# CLI (Command Line Interface)
```
Comment (ErikSchierboom) [inline exercises/shared/.docs/cli.md]: We don't support a `cli.md` file in the shared files. See [the documentation](https://exercism.org/docs/building/tracks/shared-files). Was there a page that said that we did?

Could you remove this file?

File: bin/run-tests.ps1 [hunk 3]
```diff
@@ -1,5 +1,5 @@
 $tmpRoot = [System.IO.Path]::GetTempPath()
-$exercises = Get-ChildItem -Path exercises/practice -Directory
+$exercises = Get-ChildItem -Path ../exercises/practice -Directory
```
Comment (ErikSchierboom) [inline bin/run-tests.ps1:2]: I think the idea is that one runs `pwsh ./bin/run-tests.ps1`, in which case this path is incorrect. Maybe we should make this work in both cases and do:

```suggestion
$exercises = Get-ChildItem -Path $PSScriptRoot/../exercises/practice -Directory
```

**Review by ErikSchierboom — COMMENTED** [review]
Thanks for working on this! I've left some minor comments.

File: exercises/shared/.docs/debug.md (hunk 1 above)
Comment (KalebHawkins) [reply exercises/shared/.docs/debug.md]: Yes I can remove this file. No problems there.

File: exercises/shared/.docs/cli.md (hunk 2 above)
Comment (KalebHawkins) [reply exercises/shared/.docs/cli.md]: I was using the go track accumulate exercise as kind of a template and it was in there. I will remove it.

File: bin/run-tests.ps1 (hunk 3 above)
Comment (KalebHawkins) [reply bin/run-tests.ps1:2]: That is a much better solution. I will update it.

**Comment by KalebHawkins** [comment]
@ErikSchierboom  I removed the files per request. 

@kotp I'm not really sure how I can clean up the history without reverting back everything and redoing all of it. I looked at reverting/dropping changes but not really sure how to drop just the one particular change.

**Comment by kotp** [comment]
I corrected the history here for your use, if you wanted, but I found this closed, and the branch appears to be deleted.  I am sure this is probably coming about in a different PR then.

**Comment by KalebHawkins** [comment]
@kotp Thanks kotp there are three new pull requests that all the changes in the two I closed takes place in.

* https://github.com/exercism/powershell/pull/167
* https://github.com/exercism/powershell/pull/166
* https://github.com/exercism/powershell/pull/165

Sorry about the confusion. I wasn't 100% positive how to push out some of the history and keep everything past that so I scrapped and handled my commits in a more focused manner. 

I would be interested in how you corrected the history if you don't care to share for future reference. I'm in the slack channel feel free to ping me. My user name there is Kryo.

**Pull request closed without merging**